src/w3d/renderer/w3dformat.h

```diff
@@ -171,6 +171,25 @@ inline uint8_t Alpha_Bits(WW3DFormat format)
     return 0;
 }
 
+/**
+ * Determine if the format is a compressed format.
+ */
+inline bool Is_Compressed(WW3DFormat format)
+{
+    switch (format) {
```

Comment:
Couldn't we just switch to BCn semantics and to support future texture mods with BC7 compression? Or it has to be strictly what C&C supports here?